Dockerfile

```diff
@@ -1,10 +1,23 @@
-FROM gidden/messageix-base
-
-COPY . /ixmp
-WORKDIR /
-ENV IXMP_PATH /ixmp
-ENV IXMP_R_PATH /ixmp/ixmp
-RUN pip2 install xlsxwriter xlrd
-RUN pip3 install xlsxwriter xlrd
-RUN cd /ixmp && python2 setup.py install 
-RUN cd /ixmp && python3 setup.py install
+FROM openjdk:11
+# installing python 3.7
+ENV PYTHON_VERSION=3.7.3
```

Comment:
As mentioned on Slack, the preference would be to invoke/re-use one of the scripts in the `ci/` directory to handle these steps.

This would mean that changes to the build/test environment would only need to be made in 2 places, rather than 3, and reduce the burden of maintenance.

Reply:
I am not sure what exactly we may re-use. Can we discuss this and maybe make changes together?

Reply:
The sequence of `ci/env.sh`, `ci/travis-before_install.sh`, and `ci/travis-install.sh` performs the same tasks, namely:
- Sets environment variables for Python and GAMS versions (though note that Miniconda is used on Travis, whereas Python is build from source here; we should pick one and be consistent).
- `curl`s the files.
- Installs both.
- Sets up a conda environment.
- Installs graphviz (on Travis, only for macOS).